### PR TITLE
fix(build): Migrate to gulp 4

### PR DESCRIPTION
**Pull request by negezor**
Support for Gulp 3 has been discontinued. This Pull Request adds support for Node.js older than version 11.

### CODE REVIEW

**Comment by chaubunhtha** [comment]
gulp4#5

File: gulpfile.js [hunk 1]
```diff
@@ -100,4 +97,6 @@ gulp.task('replace-demo-data',['build-demo-data'], function() {
             }
         }))
         .pipe(gulp.dest('build/assets/player/'));
```
Comment (KellyHang) [inline gulpfile.js:102]: A

**Review by hqyyds2213 — COMMENTED** [review]
厉害

**Review by Davidak643h — APPROVED** [review]

**Review by STAFFSTRONG67 — COMMENTED** [review]
e35d611d1a67f9e6ebf2fecd4fc322447a05927a

**Review by STAFFSTRONG67 — APPROVED** [review]

**Pull request merged**